src/components/30-organisms/testimonials/README.md

```diff
@@ -44,17 +21,14 @@ export default AXATestimonialsReact;
 
 ```js
 <AXATestimonialsReact
-  autorotatetime="1000"
-  autorotatedisabled="false"
-  keysenabled="true"
-  showallinline="false"
   title="The main title"
   subtitle="This is the subtitle"
+  keysenabled
 >
   <span>first slide</span>
   <div>
     <span>second slide</span>
-    <span class="author">Andrew Jackson, Advocate</span>
+    <span className="author">Andrew Jackson, Advocate</span>
```

Comment:
Use a prefix here. @LucaMele We also wanted to discuss what kind of classes we give child elements, if we even should rely on classes. AFAIR in child css we need to reference the tags. Dani can tell you abbout the discussion tomorrow.

Reply:
@danyball please use prefix and BEM always for classes. Therefore here something like: `o-testimonials__author` if is an organism or `m-testimonials__author`.

Other option is to set a attribute, which i dont like that much